unison-src/transcripts/fix3171.output.md

```diff
@@ -28,6 +28,6 @@ f x y z _ = x + y * z
   
     5 | > f 1 2 3
           ⧩
-          '(1 Nat.+ 2 Nat.* 3)
+          _ -> 1 Nat.+ 2 Nat.* 3
```

Comment:
This was an example where it was being rendered as a delay, but it was really just a lambda that ignored its argument.